internal/tracing/v2/span.go

```diff
@@ -43,6 +44,16 @@ func (s *Span) LogKV(name string, kv ...string) {
 	s.w.AddEvent(name, trace.WithAttributes(attrs...))
 }
 
+// RecordError adds an error to the span.
+func (s *Span) RecordError(err error, recordStackTrace bool) {
+	if s == nil || err == nil {
+		return
+	}
+
+	s.w.RecordError(err, trace.WithStackTrace(recordStackTrace))
```

Comment:
For documentation: https://opentelemetry.io/docs/languages/go/instrumentation/#record-errors